awscli/botocore/httpsession.py

```diff
@@ -1,46 +1,34 @@
-import logging
-import os
 import os.path
+import os
+import logging
 import socket
-import sys
 from base64 import b64encode
+import sys
 
-from urllib3 import PoolManager, ProxyManager, Timeout, proxy_from_url
-from urllib3.exceptions import (
-    ConnectTimeoutError as URLLib3ConnectTimeoutError,
-)
-from urllib3.exceptions import NewConnectionError, ProtocolError, ProxyError
-from urllib3.exceptions import ReadTimeoutError as URLLib3ReadTimeoutError
-from urllib3.exceptions import SSLError as URLLib3SSLError
+from urllib3 import PoolManager, proxy_from_url, Timeout
 from urllib3.util.retry import Retry
 from urllib3.util.ssl_ import (
-    DEFAULT_CIPHERS,
-    OP_NO_COMPRESSION,
-    OP_NO_SSLv2,
-    OP_NO_SSLv3,
-    ssl,
+    ssl, OP_NO_SSLv2, OP_NO_SSLv3, OP_NO_COMPRESSION, OP_NO_TICKET,
+    PROTOCOL_TLS, PROTOCOL_TLS_CLIENT, DEFAULT_CIPHERS,
 )
-
+from urllib3.exceptions import SSLError as URLLib3SSLError
+from urllib3.exceptions import ReadTimeoutError as URLLib3ReadTimeoutError
+from urllib3.exceptions import ConnectTimeoutError as URLLib3ConnectTimeoutError
+from urllib3.exceptions import NewConnectionError, ProtocolError, ProxyError
```

Comment:
Is there a reason we're unordering the imports? Happy to help get them realigned after the old commit if needed.

Reply:
That was a blanket copy from `botocore`s current `httpsession.py`, I can adjust it!

Reply:
Reverted these changes! Should be good now.